modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -164,16 +164,16 @@ const applyAttributes = (editor: Editor, node: Element, forcedRootBlockAttrs: Re
 const setForcedBlockAttrs = (editor: Editor, node) => {
   const forcedRootBlockName = Options.getForcedRootBlock(editor);
 
-  if (forcedRootBlockName && forcedRootBlockName.toLowerCase() === node.tagName.toLowerCase()) {
+  if (forcedRootBlockName.toLowerCase() === node.tagName.toLowerCase()) {
     const forcedRootBlockAttrs = Options.getForcedRootBlockAttrs(editor);
     applyAttributes(editor, node, forcedRootBlockAttrs);
   }
 };
 
 // Wraps any text nodes or inline elements in the specified forced root block name
-const wrapSelfAndSiblingsInDefaultBlock = (editor: Editor, newBlockName, rng, container, offset) => {
+const wrapSelfAndSiblingsInDefaultBlock = (editor: Editor, newBlockName: string, rng, container, offset) => {
   let newBlock, parentBlock, startNode, node, next, rootBlockName;
-  const blockName = newBlockName || 'P';
+  const blockName = newBlockName;
```

Comment:
Is this variable needed anymore? It looks like it's redundant and could just be removed.